### ISSUE TITLE
Weird "Expected attribute" Slim::Parser::SyntaxError with Rails 5.0.2 (Ruby 2.4.1)

### ISSUE THREAD

**Issue by elquimista** [slim-template/slim-rails#144]
## Slim::Parser::SyntaxError in LandingPage#index

```
Expected attribute
  node_modules/clone/test-apart-ctx, Line 11, Column 8
    eval(decodeURIComponent(data[1]));
    ^
```

Extracted source (around line `#5`):
```
4  .row
5    = image_tag 'logo-light.png'
6  .row

```

This issue arises with slim-rails v3.1.2. When I reverted back to 3.1.1, it didn't occur.

**Comment by januszm** [comment]
"eval is evil" 😃  especially when data comes from the user, this looks like something that comes from URI ?

What is the expected behaviour?

**Comment by clthck** [comment]
I don't directly use any `eval` thing at all. That's why I think it's weird. The expected behavior is obviously displaying logo image in the topmost section of the page. This occurs with slim-rails@3.1.2 only.

**Comment by januszm** [comment]
Yeah the thing about eval was re: https://github.com/pvorb/clone/blob/master/test-apart-ctx.html

Are you using this package?

**Comment by clthck** [comment]
I'm not using that package directly (I even never heard of that package name).

**Comment by clthck** [comment]
The thing is it's just a single patch level version number difference (3.1.2 vs 3.1.1) and that's causing this awkward error.

**Comment by januszm** [comment]
@clthck we released version 3.1.3, please upgrade and re-test
